core/startos/src/service/config.rs

```diff
@@ -10,10 +10,27 @@ impl Service {
         ConfigureContext { timeout, config }: ConfigureContext,
     ) -> Result<(), Error> {
         let container = &self.seed.persistent_container;
+        let package_id = &self.seed.id;
+
         container
-            .execute::<Value>(ProcedureName::SetConfig, to_value(&config)?, timeout)
+            .execute::<SetResult>(ProcedureName::SetConfig, to_value(&config)?, timeout)
```

Comment:
Is this type still correct for the new api?

Reply:
Not sure, but it is at the moment

Reply:
So, going back to value to say that I don't care